### PR TITLE
Landscape items refactoring

### PR DESCRIPTION
**Pull request by bonndan**
removes static use of Items.java

### CODE REVIEW

File: src/main/java/de/bonndan/nivio/input/ItemDescriptions.java [hunk 1]
```diff
@@ -1,149 +1,148 @@
...
-    public static LandscapeItem pick(final String identifier, String group, final Collection<? extends LandscapeItem> items) {
+    public ItemDescription pick(final String identifier, String group) {
         if (StringUtils.isEmpty(identifier)) {
             throw new IllegalArgumentException("Identifier to pick is empty");
         }
 
-        return find(identifier, group, items).orElseThrow(() ->
-                new RuntimeException("Element '" + identifier + "' not found  in collection " + items)
+        return find(identifier, group).orElseThrow(() ->
+                new RuntimeException("Element '" + identifier + "' not found  in collection ")
```
Comment (basseur) [inline src/main/java/de/bonndan/nivio/input/ItemDescriptions.java]: remove the space before `"`.

File: src/main/java/de/bonndan/nivio/input/ItemDescriptions.java [hunk 2]
```diff
@@ -1,149 +1,148 @@
...
      * @return the or null
      */
-    public static Optional<LandscapeItem> find(FullyQualifiedIdentifier fqi, Collection<? extends LandscapeItem> items) {
-        List<LandscapeItem> found = findAll(fqi, items);
+    public Optional<LandscapeItem> find(FullyQualifiedIdentifier fqi) {
+        List<ItemDescription> found = findAll(fqi);
 
         if (found.size() > 1)
-            throw new RuntimeException("Ambiguous result for " + fqi + ": " + found + " in collection " + items);
+            throw new RuntimeException("Ambiguous result for " + fqi + ": " + found + " in collection ");
```
Comment (basseur) [inline src/main/java/de/bonndan/nivio/input/ItemDescriptions.java]: remove space before `"`.

File: src/main/java/de/bonndan/nivio/model/LandscapeItems.java [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    }
+
+    public void setItems(Set<Item> items) {
+        index = new ConcurrentIndexedCollection<>();
+        index.addAll(items);
+    }
+
+    public void add(Item item) {
+        index.add(item);
+    }
```
Comment (basseur) [inline src/main/java/de/bonndan/nivio/model/LandscapeItems.java]: maybe Objects.requireNonNull for the items? (here and elsewhere)

File: src/main/java/de/bonndan/nivio/input/ItemDescriptions.java (hunk 1 above)
Comment (bonndan) [reply src/main/java/de/bonndan/nivio/input/ItemDescriptions.java]: can be done in develop branch

File: src/main/java/de/bonndan/nivio/input/ItemDescriptions.java (hunk 2 above)
Comment (bonndan) [reply src/main/java/de/bonndan/nivio/input/ItemDescriptions.java]: can be done in develop branch

File: src/main/java/de/bonndan/nivio/model/LandscapeItems.java (hunk 3 above)
Comment (bonndan) [reply src/main/java/de/bonndan/nivio/model/LandscapeItems.java]: ends in NPE anyway, so can be done in develop branch

**Pull request merged**